### ISSUE TITLE
ApplyFormatInEditMode not working with Date

### ISSUE THREAD

**Issue by TechnologyDude** [aspnet/AspNetCore.Docs#11535]
I have followed this tutorial, and the ApplyFormatInEditMode mentioned at the beginning of the tutorial does not work for me.  The DisplayFormat works fine on the Index, Details and Delete pages, but the format is not applied on the Edit page. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 634dfb2c-2626-9ec5-35fe-2bd2f27c5e23
* Version Independent ID: 7a77b058-f005-8d68-ae46-e487c597f18e
* Content: [Razor Pages with EF Core in ASP.NET Core - Data Model - 5 of 8](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/complex-data-model?view=aspnetcore-2.2&tabs=visual-studio#feedback)
* Content Source: [aspnetcore/data/ef-rp/complex-data-model.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/data/ef-rp/complex-data-model.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
In what way does it not work? What did you use? What did you expect?

**Comment by TechnologyDude** [comment]
I'm using this annotation:
`[DisplayFormat(DataFormatString = "{0:yyyy-MM-dd}", ApplyFormatInEditMode = true)]`
On the Index, Details and Delete pages, this causes the date to display as I expected:
**2010-09-01**
However, on the Edit page, it displays in a text box as:
**9/1/2010**

**Comment by Rick-Anderson** [comment]
@TechnologyDude there is alot of chatter on this on SO, did you read those posts?

**Comment by TechnologyDude** [comment]
I did search quite extensively before I posted anything up here, and I wasn't able to find much about this issue.  I just checked again and found a post which states "The [DisplatFormat] attribute is only respected when using DisplayFor() or EditorFor()".  The tutorial said to use:
`<input asp-for="Student.EnrollmentDate" class="form-control " />`
After reading the post, I tried changing this to:
`@Html.EditorFor(model => model.Student.EnrollmentDate)`
and in either case, I still cannot get it to apply the format on the Edit page ---- except that it does properly show only the date and not the time.
